README.md

```diff
@@ -1,180 +1,113 @@
-
-### Other Common Issues
-
-For other common installation issues, please refer to the [FAQ](https://qiskit.org/documentation/metal/faq.html)
-
-### Additional steps for developers
 
-If you are planning to develop the qiskit metal codebase, you need extra packages, which you can install by running the following command instead of (or after) the previous one:
+#### Let's create a new qubit (a transmon) by creating an object of this class.
+```python
+>>> from qiskit_metal.qlibrary.qubits.transmon_pocket import TransmonPocket
+>>> q1 = TransmonPocket(design, 'Q1', options=dict(connection_pads=dict(a=dict())))
+>>> gui.rebuild()
+>>> gui.edit_component('Q1')
+>>> gui.autoscale()
 ```
-python -m pip install -r requirements-dev.txt
+#### Change options.
+```python
+>>> q1.options.pos_x = '0.5 mm'
+>>> q1.options.pos_y = '0.25 mm'
+>>> q1.options.pad_height = '225 um'
+>>> q1.options.pad_width  = '250 um'
+>>> q1.options.pad_gap    = '50 um'
+```
+#### Update the component geoemtry, since we changed the options.
+```python
+>>> gui.rebuild()
+```
+#### Get a list of all the qcomponents in QDesign and then zoom on them.
+```python
+>>> all_component_names = design.components.keys()
+>>> gui.zoom_on_components(all_component_names)
+```
+#### Closing the Qiskit Metal GUI.
+```python
+>>> gui.main_window.close()
 ```
-You'll want to also use these instructions to [setup user environment](/docs/NEW_DEVELOPER_SETUP.md)
 
+A script is available [here](https://qiskit.org/documentation/metal/tut/overview/1.1%20High%20Level%20Demo%20of%20Qiskit%20Metal.html), where we also show the overview of Qiskit Metal.
 
+## Community and Support
 
-## Authors and Citation
+#### Watch the recorded tutorials
+The streaming will also be recorded and made available [here](https://www.youtube.com/playlist?list=PLOFEBzvs-VvqHl5ZqVmhB_FcSqmLufsjb) for offline review.
 
-Qiskit Metal is the work of [many people](https://github.com/Qiskit/qiskit-metal/pulse/monthly) who contribute to the project at different levels. Metal was conceived and developed by [Zlatko Minev](zlatko-minev.com) at IBM; then co-led with Thomas McConkey. If you use Qiskit Metal, please cite as per the included [BibTeX file](https://github.com/Qiskit/qiskit-metal/blob/main/Qiskit_Metal.bib). For icon attributions, see [here](/qiskit_metal/_gui/_imgs/icon_attributions.txt).
+#### Take part in the live tutorials and discussion
+Through June 2021 we are offering live tutorials and Q&A. [Sign up](https://airtable.com/shrxQEgKqZCf319F3) to receive an invite to the upcoming sessions.  The streaming will also be recorded and made available for offline review.  Find [here](https://github.com/Qiskit/qiskit-metal/blob/main/README_Tutorials.md) more details on schedule and use the Slack channel to give us feedback and to request the most relevant content to you.
 
+#### Get help: Slack
```

Comment:
I think it may be nice  if we had a slack icon here

Reply:
Something like this at the top of the page:
[![join slack](https://img.shields.io/badge/slack-@qiskit-yellow.svg?logo=slack)](https://ibm.co/joinqiskitslack)

and something like this here:
[![join slack](https://img.shields.io/badge/slack-blue.svg?logo=slack)](https://ibm.co/joinqiskitslack)

Reply:
Thanks. Did not know how to make that one.